tests/posix/fs/prj.conf

```diff
@@ -1,6 +1,7 @@
 CONFIG_FILE_SYSTEM=y
 CONFIG_FAT_FILESYSTEM_ELM=y
 CONFIG_DISK_ACCESS_RAM=y
-CONFIG_PTHREAD_IPC=y
+CONFIG_POSIX_API=y
+#CONFIG_PTHREAD_IPC=y
```

Comment:
You probably meant to remove this `CONFIG_PTHREAD_IPC` here

Reply:
@nniranjhana: Good catch, thought I squashed that out. Pushed update.